src/tls.c

```diff
@@ -10203,6 +10501,38 @@ static int TLSX_PopulateSupportedGroups(WOLFSSL* ssl, TLSX** extensions)
         #endif
 #endif
 
+#ifdef HAVE_LIBOQS
+    ret = TLSX_UseSupportedCurve(extensions, WOLFSSL_KYBER512, ssl->heap);
```

Comment:
Does it make sense to support all of these? Can liboqs be built with some of these disabled?

Reply:
liboqs is configurable at build time to support whatever subset of algorithms you want so for example you could build with only NTRU supported.  Now that you mention it, it may make sense for us to test if algorithms are supported (using `OQS_KEM_alg_is_enabled`) with the current liboqs before adding them to SupportedGroups, as the handshake will fail for these algorithms anyways.

Reply:
These are all Round 3 finalist algorithms.  Actually, we're missing Classic McEliece so we should add that as well.  That can be a separate PR. 

Another thing we might want to consider is adding the alternate candidates as well. We can talk about this in the engineering meeting.

Reply:
Classic McEliece was intentionally excluded as the public key size excludes TLS' max of 65535 bytes (their public keys start at 261120 bytes).  It won't be impossible to include, but will take some more effort than the other Round 3 finalists.

Reply:
HA!! I've taken off too much time away from work because you're right and I should have known that because I didn't do it for my last product as well!!  Thanks @kabuobeid